File: app/src/main/java/com/jakubwyc/mysecretvault/repository/RealmTreasure.java
package com.jakubwyc.mysecretvault.repository;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class RealmTreasure extends RealmObject {

    @Required
    public String text;
    @PrimaryKey
    public long date;

}

File: app/src/main/kotlin/com/jakubwyc/mysecretvault/repository/RealmTreasure.kt
package com.jakubwyc.mysecretvault.repository

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

open class RealmTreasure : RealmObject() {

    @Required
    var text: String = ""
    @PrimaryKey
    var date: Long = 0

}
